src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/TraceListener.cs

```diff
@@ -42,7 +42,10 @@ protected TraceListener(string? name)
             _listenerName = name;
         }
 
-        public StringDictionary Attributes => _attributes ??= new StringDictionary();
+        public StringDictionary Attributes
+        {
+            get => _attributes ??= new StringDictionary();
+        }
```

Comment:
Isn't that the longer version of what was there before?

Reply:
Yep. Reverted. Not sure how that happened.